### ISSUE TITLE
Session request notification doesn't handle apostrophe correctly

### ISSUE THREAD

**Issue by ietf-svn-bot** [ietf-tools/datatracker#934]
`owner:rcross` `type_defect`   |    by rcross@amsl.com

___

Example:

A new meeting session request has just been submitted by Karen
O&#39;Donoghue, a chair of the jose working group.

___
_Issue migrated from trac:934 at 2022-03-04 02:41:15 +0000_

**Comment by ajeanmahoney** [comment]
FYI, here are more details on what this bug looks like -- I checked this out with dt-main.dev.ietf.org by trying to request an interim session as Karen. The sandbox accepted the request and said that a meeting session request had been submitted (i.e., no errors). However, in the message that is emitted, the code `&#x27;` is used in place of the apostrophe in Karen's name (see the `Session Requester:` line): 

```
An interim meeting for jose has been approved or does not require approval and is ready for announcement.

Use this link to officially announce the meeting:
https://datatracker.ietf.org/meeting/interim/request/interim-2024-jose-01



---------------------------------------------------------
Working Group Name: Javascript Object Signing and Encryption
Area Name: Security Area
Session Requester: Karen O&#x27;Donoghue

Meeting Type: Virtual Meeting

Session 1:

Date: 2024-08-30
Start Time: 12:00 America/Vancouver
Duration: 01:00
Remote Participation Information: Remote participation information will be obtained at the time of approval
Agenda Note: testing https://github.com/ietf-tools/datatracker/issues/934
```

**Comment by jennifer-richards** [comment]
Closed by #8007
